src/tests/system/tests/test_ipa_trusts.py

```diff
@@ -1,5 +1,5 @@
 """
-Identity of trusted users and groups.
+IPA Trusts.
 
 :requirement: IDM-SSSD-REQ: Testing SSSD in IPA Provider
```

Comment:
Are we keeping this requirement?

Reply:
I don't know. I want to rename the requirements to match the categories, I don't know what the level of effort involved. I'll all for 'the path of least resistance'.

Reply:
I am okay to keep as of now.